tests/integration/helpers.py

```diff
@@ -445,11 +445,34 @@ async def get_address(ops_test: OpsTest, app_name=APP_NAME, unit_num=0) -> str:
     return address
 
 
-def balancer_is_running(model_full_name: str | None) -> bool:
+def balancer_is_running(model_full_name: str | None, app_name: str) -> bool:
```

Comment:
**suggestion** This method should probably live on `workload`

Reply:
Marc added something similar [in his own PR](https://github.com/canonical/kafka-operator/pull/216/files#diff-08c451fa9005a5da5cae9a06ac08e11c96fa7f50c7c5aaac635952ebcd11e172)

Reply:
nvm, for some reason I thought I was looking at charm code here instead of tests. My comment is irrelevant.